cmd/terramate/main.go

```diff
@@ -17,17 +17,47 @@ package main
 import (
 	"log"
 	"os"
+	"runtime/debug"
 
+	"github.com/mineiros-io/terramate"
 	"github.com/mineiros-io/terramate/cmd/terramate/cli"
+	"golang.org/x/mod/semver"
 )
 
 func main() {
 	wd, err := os.Getwd()
 	if err != nil {
 		log.Fatalf("failed to get process working dir: %v", err)
 	}
+	defineVersion()
 	err = cli.Run(wd, os.Args[1:], os.Stdin, os.Stdout, os.Stderr)
 	if err != nil {
 		log.Fatal(err)
 	}
 }
+
+func defineVersion() {
+	const (
+		defaultVersion = "v0.0.0-devel"
```

Comment:
why `v`?

Reply:
because Go uses versions with v prefix, calling: debug.ReadBuildInfo() when you have an actual release will return a version with the v prefix, which then we need to remove, this default is a default for the Go version when it is not available (it is not always available), hence the v.

Reply:
actually on this case it is inconsistent indeed, when shifting from Go semver and actual semver I got confused, this one never gets the v prefix removed and is wrong, will fix. thanks for the catch :100: